tests/test_events/test_events.py

```diff
@@ -2138,3 +2138,28 @@ def test_start_replay_send_to_log_group():
     event_replay["resources"].should.be.empty
     event_replay["detail"].should.equal({"key": "value"})
     event_replay["replay-name"].should.equal("test-replay")
+
+
+@mock_events
+def test_create_and_list_connections():
```

Comment:
We should have a separate list_connections test to verify it works without creating any connections first